keras_nlp/utils/text_generation.py

```diff
@@ -218,6 +218,7 @@ def beam_search(
     from_logits=False,
     end_token_id=None,
     pad_token_id=0,
+    return_all_beams_and_probs=False,
```

Comment:
We can probably just leave the old version as is right @chenmoneygithub ? This will eventually be deprecated for the new sampler API.